### PR TITLE
[DE] Setup

### PR DESCRIPTION
**Pull request by markdermark**

### CODE REVIEW

**Comment by qwrtln** [comment]
Pictures, please.

**Comment by qwrtln** [comment]
po4a check

File: translations/setup.tex/de.po [hunk 1]
```diff
@@ -79,18 +81,18 @@ msgid ""
...
+"    \\item[c)]7 x Town Building Tile\n"
+"    \\item[d)]1 x Town Board\n"
+"    \\item[e)]7 x Double-sided Unit Card\n"
+"    \\item[f)]3 x Hero-specific Specialty Card (of the chosen Hero)\n"
+"    \\item[g)]1 x Hero-specific Ability Card (of the chosen Hero)\n"
+"    \\item[h)]20 x Faction Cube\n"
+"    \\item[i)]1 x Build Token\n"
+"    \\item[j)]1 x Population Token\n"
+"    \\item[k)]1 x Spell Book Token\n"
+"    \\item[l)]3 x Movement Tokens\n"
```
Comment (qwrtln) [inline translations/setup.tex/de.po:95]: You changed the wrong part. Please keep the original intact.

**Comment by markdermark** [comment]
po4a commit

**Comment by qwrtln** [comment]
Nice, now you broke it 😅

**Review by qwrtln — CHANGES_REQUESTED** [review]
Fix the fuzzy translation

**Comment by markdermark** [comment]
sry i got confused their, i close this pull and start a new one.

**Comment by qwrtln** [comment]
No need, don't close. Just update the file.

**Pull request closed without merging**